app/config/parameters.yml.dist

```diff
@@ -31,3 +31,8 @@ parameters:
 
     recaptcha_site_key: ~
     recaptcha_secret_key: ~
+
+    consumer_key: ~         #twitter
```

Comment:
Please change them to twitter_consumer_key etc. Because maybe we add a consumer_key for Pinterest soon, which can be named pinterest_consumer_key. That is also more consistent with recaptcha_site_key and recaptcha_secret_key